tests/simple.rs

```diff
@@ -26,6 +26,10 @@ fn simple_get() {
         "GET http://localhost:{port}/simple_get/{{{{id}}}}",
         port = server.port()
     ));
+    let script_file_without_method = create_file(&format!(
```

Comment:
In order to keep this test truly `simple_get` let's leave only 1 get request inside the body. 

To test the fallback on `GET` method, we can either:
1. remove explicit `GET` method in `simple_get` test 
```rust
    let script_file = create_file(&format!(
        "http://localhost:{port}/simple_get/{{{{id}}}}",
        port = server.port()
    ));
```
2. Add this new test in the `multi.rs` file
    
Personally I think taking easiest option 1 should be fine

Reply:
I agree on the nitpick. I'm not happy with the test and my change on it tbh. I've implemented your first approach as I think it's good enough for now.
My problem with this: I've played a bit around with the parsing and found for example that the parser is really strict about whitespaces compared to jetbrains http client. I'd like to have proper unit tests that focuses on the parsing alone to make it easier to cover mutliple test cases. I'll create a separate issue for this.